modules/installation-configuration-parameters.adoc

```diff
@@ -964,6 +972,11 @@ This property is deprecated. To use a flavor as the default for all machine pool
 |String, for example `m1.xlarge`.
 |====
 
+. If the machine pool defines `zones`, the count of types can either be a single item or match the number of items in `zones`. For example, the count of types cannot be 2 if there are 3 items in `zones`.
+
+. Because this property is deprecated, the installer converts the provided value into a list of string with one item, and populates the `types` field with this list item.
```

Comment:
This note should refer to the `type` field, but I see no reference to it. Is it more evident when the document is formatted?

Reply:
You can check the preview link to see how it renders. It is added as a footnote with reference.

This mechanism is explained to let users know that `type` is still supported and converted to `types`. This also helps them know that they need not redo any work if they have any existing references to `type`. 
Do you think it needs to be said differently? Like `Existing references to this field need not be changed` ??

Reply:
>You can check the preview link to see how it renders. It is added as a footnote with reference.

Ok, checked! Looks perfect.

>This also helps them know

That makes sense, I just hope we're not going too much into the weeds. But that's your call, not mine :)

Reply:
Sure. I will reword this to touch upon the conversion and just let users know that the values under `type` is converted to `types` (without the single list item explanation)

Reply:
@pierreprinetti I have reworded the current statement. Can you have a look and let me know it's ok?

I have made all the suggested changes. Let me know if there are any more suggestions. If not, can you give an lgtm?